app/commands/source/reparent_category_command.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Source
+  class ReparentCategoryCommand < ApplicationCommand
+    usage do
+      no_command
+    end
+
+    keyword :category do
+      desc "The target category ID"
+      required
+      validate { _1 =~ Category::ID_REGEX }
+    end
+
+    keyword :parent do
+      desc "The new parent category ID"
+      required
+      validate { _1 =~ Category::ID_REGEX }
+    end
+
+    def execute
+      find_category!
+      find_parent!
+
+      frame("Reparenting category") do
+        logger.headline("Category: #{@category.name}")
+        logger.headline("Parent: #{@parent.name}")
+
+        update_category!
+        update_data_files!
```

Comment:
Yes very — I've held off on mappings _for now_ because there are several other mappings PRs in flight.

I'd actually love @jeanrick and team to tackle this particular piece as well

Reply:
One thing that occurred to me is that we might also need to delete invalid mappings in addition to creating new ones 🤔